CMakeLists.txt

```diff
@@ -33,7 +33,14 @@ set(COMMON_PROJECT_DOMAIN eu.humanbrainproject)
 
 include(Common)
 
-include(FindPackages)
+if(LINUX)
+  common_package(avahi-client)
+else()
```

Comment:
Is DNSSD available in Windows? Otherwise use elseif(APPLE) as in the original code.